### PR TITLE
improve CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by TomasTomecek**
I require a review from @lachmanfrantisek since I made some statements which may not be true.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 Before creating bug reports, please check a [list of known issues](https://github.com/user-cont/colin/issues) to see
-if the problem has already been reported.
+if the problem has already been reported (or fixed in a master branch).
 
 If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/user-cont/colin/issues/new).
-Be sure to include a **descriptive title, clear description and a package version** and please include **as many details as possible** to help maintainers reproduce
-the issue and resolve it faster.
+Be sure to include a **descriptive title and a clear description**. Ideally, please provide:
+ * version of colin you are using
+ * version of conu library
```
Comment (lachmanfrantisek) [inline CONTRIBUTING.md]: Maybe also `dockerfile-parse`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -54,14 +63,15 @@ class FooBarLabelCheck(LabelCheck):
...
-- `value_regex` means regular expression to check label.
+Let's go through the list of keyword arguments:
+
+- `name` — name of the check, this will be used to reference this check inside a ruleset.
+- `message` — error message when the check fails.
+- `description` gives information in colin's output on what this check does.
+- `reference_url` — URL with even more info about the check. Ideally link to guidelines.
+- `tags` — used to filter checks to run (`colin check -t foo` would run checks which have tag `foo`).
+- `label` is a keyword argument specific to `LabelCheck` class.
+- `required` — global keyword for `Check` class. If the check fails and this is true, the check is marked as failed. If this is false and the check fails, it is a warning.
```
Comment (lachmanfrantisek) [inline CONTRIBUTING.md]: This is currently not global -- now, not all checks use that, but maybe it makes sense to make it global.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -25,6 +26,14 @@ When you are creating an enhancement issue, **use a clear and descriptive title*
 and **provide a clear description of the suggested enhancement**
 in as many details as possible.
 
+## How to contribute code to colin
```
Comment (lachmanfrantisek) [inline CONTRIBUTING.md]: Good point, I like this part.

File: CONTRIBUTING.md (hunk 1 above)
Comment (TomasTomecek) [reply CONTRIBUTING.md]: +1

Comment (TomasTomecek) [reply CONTRIBUTING.md]: I'll also make them clickable links.

File: CONTRIBUTING.md (hunk 2 above)
Comment (TomasTomecek) [reply CONTRIBUTING.md]: so it's specific for LabelCheck? will update then

**Comment by TomasTomecek** [comment]
updated

File: CONTRIBUTING.md (hunk 1 above)
Comment (lachmanfrantisek) [reply CONTRIBUTING.md]: That's useful.

File: CONTRIBUTING.md (hunk 2 above)
Comment (lachmanfrantisek) [reply CONTRIBUTING.md]: Hmm, it's messier than I thought:
- InstructionCheck
- DockerfileLabelCheck
- EnvCheck
- LabelCheck

**Comment by phracek** [comment]
I would be glad, if this pull request is merged tomorrow

**Comment by TomasTomecek** [comment]
rebased

**Comment by lachmanfrantisek** [comment]
@TomasTomecek  Thanks!!

**Pull request merged**